crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1710,6 +1732,22 @@ impl ScheduleBuildSettings {
     }
 }
 
+/// list of [`ComponentId`]'s to ignore when reporting ambiguity conflicts between systems
+#[derive(Resource, Default, Clone)]
+pub struct IgnoreSchedulingAmbiguities(Vec<ComponentId>);
```

Comment:
This should have two distinct fields: one for resources, and one for components. Otherwise the API below doesn't make sense.

Reply:
The difference between one or two fields can't be observed in the API, so this seems alright to me. Only reason I could think to split it would be for a nicer `Debug` impl.

Reply:
I'm not a fan of splitting it up, makes the logic for filtering more complicated.